src/style.css

```diff
@@ -1 +1,73 @@
 /* Styles go here */
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  position: sticky;
+  top: 0;
+  padding-left: 50px;
+  padding-right: 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+}
+
+.navList {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.navLink {
+  padding: 22px 0;
+  display: block;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  line-height: 16px;
+}
+
+.is-active {
+  color: #00acdc;
+  padding-bottom: 0;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  border: 2px solid #00acdc;
+  background: #00acdc;
+  border-radius: 8px;
+  margin-top: 18px;
```

Comment:
what if I will increase padding for links? it will look like this 
![image](https://user-images.githubusercontent.com/64437769/142429643-b49b8ca1-0e31-4375-b13d-680fe6111f02.png)

use position: absolute here